doc/workshops/scientific-computing.rst

```diff
@@ -2,4 +2,800 @@
+
+.. code-block:: python
+
+   dog_or_cat_col_names = list(dog_or_cat_pivot.columns)
+   dog_or_cat_pivot.columns = ["A", "B", "C", "D"]
+   dog_or_cat_pivot.dropna(how="all", inplace=True)
+
+.. note::
+
+   In the variable ``dog_or_cat_col_names`` we have stored the names of the values in order. This implies that: A = Both, B = Cats, C = Dogs, and D = Neither.
+
+To perform ANOVA, we are going to use a Python library called "stats", in particular the ``f_oneway`` function. You can find more information about this library by typing ``stats`` in the Help Pane or learn more about the function by typing ``stats.f_oneway``.
+
+.. image:: images/workshop-2/help-pane-for-stats.f_oneway.png
+   :alt: stats.f_oneway() help in Spyder's Help pane
+
+The help notes that this function requires an array-like with the measurements for each group as an input (in our case, one sample for A, one for B, one for C, and one for D). Since we have empty cells in each column, let's remove them from each column (the ``f_oneway`` function does not support missing values).
+
+.. code-block:: python
+
+   dog_or_cat_samples = [dog_or_cat_pivot[col].dropna() for col in dog_or_cat_pivot]
+
+In the above line of code we are creating a list with 4 elements: the test results of each column A, B, C, and D, without the missing values (they are removed with ``dropna()``).
+
+
+Run ANOVA
+~~~~~~~~~
+
+Now, let's run ANOVA and store the output in two variables: ``f_value`` and ``p_value``. Notice that since ``dog_or_cat_samples`` is a list, we must pass the argument with an asterisk (``*dog_or_cat_samples``).
+
+.. code-block:: python
+
+   f_value, p_value = stats.f_oneway(*dog_or_cat_samples)
+
+With the help of ``stats.f_oneway`` in the Help Pane you can read a description of the output of this function, some interesting notes about the test, and some references that you can use to better understand the nature of this analysis.
+
+.. important::
+
+   In the Help pane you will read that ANOVA requires certain assumptions to be satisfied. In this workshop we have not checked those assumptions because our goal is merely to show some Spyder functions that make the research work easier. Thus, the results obtained in this example with ANOVA should not be taken too rigorously.
+
+In the Variable Explorer you can find these two new variables: ``f_value`` and ``p_value``. ``p_value`` (0.6275) is well over 0.05 , which gives support to the null hypothesis (detracting support from our hypothesis). To know if the F value obtained is greater than the theoretically expected value, a **F critical value** must be calculated. This can be estimated using the ``stats`` library, a significance level (*q* ), and the degrees of freedom (*df* ) for the number of groups and the number of observations:
```

Comment:
```suggestion
In the Variable Explorer you can find these two new variables: ``f_value`` and ``p_value``. ``p_value`` (0.6275) is well over 0.05, which gives support to the null hypothesis (detracting support from our hypothesis). To know if the F value obtained is greater than the theoretically expected value, a **F critical value** must be calculated. This can be estimated using the ``stats`` library, a significance level (*q* ), and the degrees of freedom (*df* ) for the number of groups and the number of observations:
```